README.md

```diff
@@ -1,2 +1,58 @@
 # OctolinerPi
-Octoliner library for Raspberry Pi
+
+A library for Raspberry Pi to interface with the Amperka [Octoliner](https://my.amperka.com/modules/octoliner) 8-channel line sensor.
+
+## Installation
+
+Open the terminal on your Raspberry Pi and use `pip` to install the library:
+
+```shell
+pip install octoliner
+```
+
+If you haven’t enabled I²C support in your Raspbian Linux yet, run the configuration tool to turn it on:
+
+```shell
+sudo raspi-config
+```
+
+Then: Interfacing Options → I2C → Yes (enable) → Yes (autoload) → \<Finish\> → Yes (reboot). The setting preserves across reboots.
+
+```console
+$ octoliner
```

Comment:
After installing the Oсtoliner library, the `octoliner` tool is not added to `/usr/bin/`. This will mislead the user. It’s better to transfer this section after the example, and run as `python3 example.py`.